packages/reports/tests/integration/components/dimension-bulk-import-test.js

```diff
@@ -237,7 +237,7 @@ module('Integration | Component | Dimension Bulk Import', function(hooks) {
 
     //Remove First Valid Pill, item removed depends on the ordering
     run(() => {
-      $('.items-list:first .item:first button').click();
+      $('.items-list:first .item:first .remove-pill').click();
```

Comment:
Do we need to use jquery here?

Reply:
both. Jquery is fine when we need to do content selectors, but otherwise we don't need them. And we should be using the event helpers rather than jquery event triggers.